src/index.ts

```diff
@@ -1,495 +1,18 @@
-    return tag.createTag(this.url)
-  }
-
-  /**
-   * Retrieve tag information from Bee node
-   *
-   * @param tag UID or tag object to be retrieved
-   */
-  retrieveTag(tagUid: number | Tag): Promise<Tag> {
-    return tag.retrieveTag(this.url, tagUid)
-  }
-
-  /**
-   * Pin file with given reference
-   *
-   * @param reference Bee file reference
-   */
-  pinFile(reference: Reference): Promise<BeeResponse> {
-    return pinning.pinFile(this.url, reference)
-  }
-
-  /**
-   * Unpin file with given reference
-   *
-   * @param reference Bee file reference
-   */
-  unpinFile(reference: Reference): Promise<BeeResponse> {
-    return pinning.unpinFile(this.url, reference)
-  }
-
-  /**
-   * Pin collection with given reference
-   *
-   * @param reference Bee collection reference
-   */
-  pinCollection(reference: Reference): Promise<BeeResponse> {
-    return pinning.pinCollection(this.url, reference)
-  }
-
-  /**
-   * Unpin collection with given reference
-   *
-   * @param reference Bee collection reference
-   */
-  unpinCollection(reference: Reference): Promise<BeeResponse> {
-    return pinning.unpinCollection(this.url, reference)
-  }
-
-  /**
-   * Pin data with given reference
-   *
-   * @param reference Bee data reference
-   */
-  pinData(reference: Reference): Promise<BeeResponse> {
-    return pinning.pinData(this.url, reference)
-  }
-
-  /**
-   * Unpin data with given reference
-   *
-   * @param reference Bee data reference
-   */
-  unpinData(reference: Reference): Promise<BeeResponse> {
-    return pinning.unpinData(this.url, reference)
-  }
-
-  /**
-   * Send to recipient or target with Postal Service for Swarm
-   *
-   * @param topic Topic name
-   * @param target Target message address prefix
-   * @param data Message to be sent
-   * @param recipient Recipient public key
-   *
-   */
-  pssSend(
-    topic: string,
-    target: AddressPrefix,
-    data: string | Uint8Array,
-    recipient?: PublicKey,
-  ): Promise<BeeResponse> {
-    return pss.send(this.url, topic, target, data, recipient)
-  }
-
-  /**
-   * Subscribe to messages with Postal Service for Swarm
-   *
-   * @param topic Topic name
-   * @param handler Message handler interface
-   *
-   * @returns Subscription to a given topic
-   */
-  pssSubscribe(topic: string, handler: PssMessageHandler): PssSubscription {
-    const ws = pss.subscribe(this.url, topic)
-
-    let cancelled = false
-    const cancel = () => {
-      if (cancelled === false) {
-        cancelled = true
-        // although the WebSocket API offers a `close` function, it seems that
-        // with the library that we are using (isomorphic-ws) it doesn't close
-        // the websocket properly, whereas `terminate` does
-        ws.terminate()
-      }
-    }
-
-    const subscription = {
-      topic,
-      cancel,
+import { Bee } from './bee'
+import { BeeDebug } from './bee-debug'
+
+export * as Utils from './utils'
+export * from './types/index'
```

Comment:
I wonder if we need to export all the types, but over-exporting is better than not exporting something :)

Reply:
if one type shouldn't be exported, then shoudln't be exported by default or at least should be placed in different folder than this common one.

Reply:
Just to stay consistent :)
```suggestion
export * from './types'
```
> if one type shouldn't be exported, then shouldn't be exported by default 

Yeah I could get behind this